blockchain/blockchain-wallet-v4-frontend
# Feat/earn

## Description (optional)
* Add staking to earn history
* Add transactions in progress (custodial only) to staking account overview

Loom: https://www.loom.com/share/dc6580d36ccc472591fb4c430a99bed7

## Testing Steps (optional)
#### Requirements
* Have an account in Argrentina
* Have ETH in a custodial account

#### Testing transactions in progress
* Go to /earn
* Click on stake/manage
* If you click mange this will take you to account overview, just click stake at the bottom
* If you clicked stake this will take you to deposit flow
* Stake your eth (min 1USD)
* After you stake your eth, you should see the "transaction in progress"

#### Testing history
* Make sure you have staked at least one time
* go to /earn/history
* scroll down until you see "ETH Staking Account"
* Click download and wait for the button to change to Save Report
* Open the CSV file and you should see a product column and it should have both staking and rewards.
